cli/pack/systemd_dir_test.go

```diff
@@ -0,0 +1,309 @@
+package pack
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"github.com/otiai10/copy"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/tarantool/tt/cli/config"
+	"github.com/tarantool/tt/cli/pack/test_helpers"
+)
+
+func Test_initSystemdDir(t *testing.T) {
+	baseTestDir := t.TempDir()
+
+	prefixToUnit := filepath.Join("etc", "systemd", "system")
+	fakeCfgPath := "/path/to/cfg"
+
+	var (
+		test1Dir = "test_default_template_values"
+		test2Dir = "test_default_template_partly_defined_values"
+		test3Dir = "test_default_template_fully_defined_values"
+		appDir   = "app"
+	)
+	testDirs := []string{
+		filepath.Join(test1Dir, appDir),
+		filepath.Join(test2Dir, appDir),
+		filepath.Join(test3Dir, appDir),
+	}
+
+	err := test_helpers.CreateDirs(baseTestDir, testDirs)
+	require.NoError(t, err)
+
+	require.NoError(t, copy.Copy(filepath.Join("testdata", "partly-defined-params.yaml"),
```

Comment:
Why file are copied one-by-one? copy.Copy is able to copy fs tree.

Reply:
Fixed.